pastel/reg_ticket.go

```diff
@@ -0,0 +1,71 @@
+package pastel
+
+// RegTickets is a collection of RegTicket
+type RegTickets []RegTicket
+
+// RegTicket represents pastel registration ticket.
+type RegTicket struct {
+	Height        int           `json:"height"`
+	TXID          string        `json:"txid"`
+	RegTicketData RegTicketData `json:"ticket"`
+}
+
+// RegTicketData is Pastel Registration ticket structure
+type RegTicketData struct {
+	Type          string    `json:"type"`
+	ArtistHeight  int       `json:"artist_height"`
+	Key1          string    `json:"key1"`
+	Key2          string    `json:"key2"`
+	IsGreen       bool      `json:"is_green"`
+	StorageFee    int       `json:"storage_fee"`
+	TotalCopes    int       `json:"total_copes"`
+	Royalty       int       `json:"royalty"`
+	Version       int       `json:"version"`
+	ArtTicket     []byte    `json:"art_ticket"`
+	ArtTicketData ArtTicket `json:"-"`
+}
+
+// ArtTicket is Pastel Art Ticket
+type ArtTicket struct {
+	Version       int       `json:"version"`
+	Blocknum      int       `json:"blocknum"`
+	Author        []byte    `json:"author"`
+	DataHash      []byte    `json:"data_hash"`
```

Comment:
check http://pastel.wiki/en/Architecture/Components/TicketStructures then this should be `block_hash`.
Could you confirm?